content/blog/2022-01-15-moving-local-experiments-to-the-cloud-with-iterative-terraform-provider.md

```diff
@@ -0,0 +1,158 @@
+---
+ title: Moving Local Experiments to the Cloud with Iterative Terraform Provider
+ date: 2022-01-15
+ description: >
+   In this tutorial you'll learn how you can move a locally run machine learning
+   experiment to a remote machine on AWS with the help of Iterative Terraform Provider.
+ descriptionLong: >
+   In this tutorial you'll learn how you can move a locally run machine learning
+   experiment to a remote machine on AWS with the help of Iterative Terraform Provider.
+ picture: 2022-01-15/massimiliano-latella-6ufBhNungOk-unsplash.jpg
+ author: maria_khalusova
+#  todo: commentsUrl: 
+ tags:
+   - MLOps
+   - Iterative Terraform Provider
+   - AWS
+ ---
+
+There's a number of good reasons one might want to train a machine learning model locally. It's quick and easy to set up a new project on a local machine, you don't have to worry about how much money you'll spend while using this compute resource, your data may be small or you start with a sample, you may want to start with simple models initially, etc. On top of that, a local machine is just deeply familiar, as opposed to the multitude of available cloud services which can be intimidating unless you have a decent background in DevOps.
+
+Once you set up everything locally, and iterate through your code and experiments enough times, you may come to a point where you have a need for a more powerful compute resource to train a larger model, use more data, or both. In other words, you may need to switch from training a model locally to training it in a cloud environment. If you find yourself in this situation, I hope this tutorial will help you by showing how to provision cloud infrastructure with Terraform and run your existing training script on it. 
+
+## Getting Started
+
+For this tutorial I have picked [The BeeImage Dataset: Annotated Honey Bee Images](https://www.kaggle.com/jenny18/honey-bee-annotated-images) that contains 5,100+ bee images annotated with location, date, time, subspecies, health condition, caste, and pollen. I've downloaded the images, created a project and trained a simple CNN model locally to classify different subspecies. If you want to follow along, you can use your own data and training code, or clone [my repo from GitHub](https://github.com/MKhalusova/bees).
```

Comment:
> [my repo from GitHub](https://github.com/MKhalusova/bees).

Iterative repo is better

Reply:
Changed the wording in 49d5c91ec296ea985f46ad51ff152a410cbecbc0. Will update the link as soon as the repo is polished and moved under Iterative.

Reply:
Moved the repo to https://github.com/iterative/bees 
Still private but you should have access to it.